examples/connector/golang/golang_connector/main.go

```diff
@@ -36,7 +36,7 @@ func (s *server) Update(in *pb.UpdateRequest, stream pb.Connector_UpdateServer)
 	state := MyState{}
 	json.Unmarshal([]byte(state_json), &state)
 
-	message := fmt.Sprintf("config: %s, selection: %s, state_json: %s, mystate: %s", config, selection, state_json, state)
+	message := fmt.Sprintf("config: ", config, "selection: ", selection, "state_json: ", state_json, "mystate: ", state)
 	LogMessage(INFO, message)
 
```

Comment:
The formatter doesn't accept json value for `%s` argument. Reverted it to the old format.